doc/langref.html.in

```diff
@@ -5010,6 +5010,11 @@ fn cmpxchgWeakButNotAtomic(comptime T: type, ptr: *T, expected_value: T, new_val
               This is usually <code>src/main.zig</code> but depends on what file is built.
           </li>
       </ul>
+      <p>
+      {#syntax#}@import{#endsyntax#} can also import ZON at compile time.
+      Example: {#syntax#}const foo: Foo = @import("foo.zon");{#endsyntax#}
+      Right now a known result type is required.
```

Comment:
Since https://github.com/ziglang/zig/pull/22907 has been merged, you can import ZON without providing a result type.